documentation/modules/configuring/proc-managing-node-pools-ids.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-managing-node-pools-ids-{context}']
+= (Preview) Assigning IDs to node pools for scaling operations
+
+[role="_abstract"]
+This procedure describes how to use annotations for advanced node ID handling by the Cluster Operator when performing scaling operations on node pools. 
+You specify the node IDs to use, rather than the Cluster Operator using the next ID in sequence.
+Management of node IDs in this way gives greater control.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode.
+
+To add a range of IDs, you assign the following annotations to the `KafkaNodePool` resource:
+
+* `strimzi.io/next-node-ids` to add a range of IDs for scaling up
+* `strimzi.io/remove-node-ids` to add a range of IDs for scaling down
+
+You can specify an array of individual node IDs, ID ranges, or a combination of both.
+For example, you can specify the following range of IDs: `[0, 1, 2, 10-20, 30]` for scaling up the Kafka node pool.
+This format allows you to specify a combination of individual node IDs (`0`, `1`, `2`, `30`) as well as a range of IDs (`10-20`).
+
+In a typical scenario, you might specify a range of IDs for scaling up and a single node ID to remove a specific node when scaling down.
+
+In this procedure, we add the scaling annotations to node pools as follows:
+
+* `pool-a` is assigned a range of IDs for scaling up 
+* `pool-b` is assigned a range of IDs for scaling down
+
+During the scaling operation, IDs are used as follows: 
+
+* Scale up picks up the lowest available ID in the range for the new node.
+* Scale down removes the node with the highest available ID in the range. 
+
+If there are gaps in the sequence of node IDs assigned in the node pool, the next node to be added is assigned an ID that fills the gap.
+
+The annotations don't need to be updated after every scaling operation.
+Any unused IDs are still valid for the next scaling event. 
+
+The Cluster Operator allows you to specify a range of IDs in either ascending or descending order, so you can define them in the order the nodes are scaled. 
+For example, when scaling up, you can specify a range such as [`1000-1999`], and the new nodes are assigned the next lowest IDs: `1000`, `1001`, `1002`, `1003`, and so on. 
+Conversely, when scaling down, you can specify a range like [`1999-1000`], ensuring that nodes with the next highest IDs are removed: `1999`, `1998`, `1997`, `1996`, and so on.
```

Comment:
I wonder if this should have a better example to clarify that it will remove the highest used node ID. E.g. with a node pool with IDs `1000`, `1001`, `1002`, `1003`, it will remove them as `1003`, `1002` etc. I think that might be more clear than starting with `1999` which will in this example be rarely used.

Reply:
Yeah. I've updated.